docker-compose.test.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  integration-runner:
```

Comment:
currently this docker-compose file is just running the tests from `data_management/opensearch_indexer` so maybe we move it/ rename it accordingly, and likewise rename this service name from `integration-runner`  to `opensearch_indexer_tests` or similar